x-pack/platform/plugins/private/upgrade_assistant/public/application/components/overview/overview.tsx

```diff
@@ -67,45 +77,69 @@ export const Overview = withRouter(({ history }: RouteComponentProps) => {
     });
   };
 
+  const versionTooltipContent = () => {
+    return currentVersion >= MIN_VERSION_TO_UPGRADE_TO_LATEST ? (
+      <></>
+    ) : (
+      <EuiToolTip
+        position="right"
+        content={
+          <FormattedMessage
+            id="xpack.upgradeAssistant.overview.latestMinVersionTooltip"
+            defaultMessage="Upgrading to v{latestVersion} requires v{minVersionToUpgradeToLatest}.
+            An alternative upgrade path is v{alternativeVersionUpgradeFrom} to v{alternativeVersionUpgradeTo} (when available)."
+            values={{
+              latestVersion: LATEST_VERSION,
+              minVersionToUpgradeToLatest: MIN_VERSION_TO_UPGRADE_TO_LATEST,
+              alternativeVersionUpgradeFrom: ALTERNATIVE_VERSION_UPGRADE_FROM,
+              alternativeVersionUpgradeTo: ALTERNATIVE_VERSION_UPGRADE_TO,
+            }}
+          />
+        }
+      >
+        <EuiIcon type="iInCircle" size="s" />
+      </EuiToolTip>
+    );
+  };
```

Comment:
nit: I think its better to use an early return than a conditional renderer with an empty fragment

```suggestion
    if (currentVersion >= MIN_VERSION_TO_UPGRADE_TO_LATEST) {
      return null;
    }
    
    return (
      <EuiToolTip
        position="right"
        content={
          <FormattedMessage
            id="xpack.upgradeAssistant.overview.latestMinVersionTooltip"
            defaultMessage="Upgrading to v{latestVersion} requires v{minVersionToUpgradeToLatest}.
            An alternative upgrade path is v{alternativeVersionUpgradeFrom} to v{alternativeVersionUpgradeTo} (when available)."
            values={{
              latestVersion: LATEST_VERSION,
              minVersionToUpgradeToLatest: MIN_VERSION_TO_UPGRADE_TO_LATEST,
              alternativeVersionUpgradeFrom: ALTERNATIVE_VERSION_UPGRADE_FROM,
              alternativeVersionUpgradeTo: ALTERNATIVE_VERSION_UPGRADE_TO,
            }}
          />
        }
      >
        <EuiIcon type="iInCircle" size="s" />
      </EuiToolTip>
    );
  };
```